code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -373,3 +373,19 @@
 	cold_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	min_cold_protection_temperature = T0C - 20
 	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/punkvest
+	name = "punk vest"
+	desc = "What are you looking punk?"
+	icon_state = "punkvest"
+	item_state = "punkvest"
+	armor = list(
+		melee = 10,
+		bullet = 10,
+		energy = 0,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+	body_parts_covered = UPPER_TORSO
+	flags_inv = HIDEJUMPSUIT
```

Comment:
Why would it hide your jumpsuit?

Reply:
So vest can be on top of jumpsuit.

Reply:
Suits are already on top of jumpsuits. Having this flag would just make the jumpsuit invisible while wearing the jacket.

You're making a jacket that makes people naked.

Reply:
Okay, i will remove this flag.

Reply:
Done